examples/polypharmacy_gripnet/config.py

```diff
@@ -1,42 +1,56 @@
+from typing import List, Tuple
+
 from yacs.config import CfgNode
 
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 # Config definition
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 
 C = CfgNode()
 
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 # Dataset
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 C.DATASET = CfgNode()
 C.DATASET.ROOT = "./data"
-C.DATASET.NAME = "PoSE"
-C.DATASET.URL = "https://github.com/pykale/data/raw/main/graphs/pose.pt"
+C.DATASET.NAME = "pose"
+C.DATASET.URL = "https://github.com/pykale/data/raw/main/graphs/pose_pyg_2.pt"
 
-# ---------------------------------------------------------------------------- #
+# ---------------------------------------------------------
 # Solver
-# ---------------------------------------------------------------------------- #
+# ---------------------------------------------------------
 C.SOLVER = CfgNode()
-C.SOLVER.SEED = 2020
+C.SOLVER.SEED = 1111
 C.SOLVER.BASE_LR = 0.01
-C.SOLVER.LR_MILESTONES = [30, 60, 90]
-C.SOLVER.LR_GAMMA = 0.1
-C.SOLVER.MAX_EPOCHS = 5
-C.SOLVER.WARMUP = False
-C.SOLVER.WARMUP_EPOCHS = 100
-
-# ---------------------------------------------------------------------------- #
-# GripNet configs
-# ---------------------------------------------------------------------------- #
-C.GRIPN = CfgNode()
-C.GRIPN.GG_LAYERS = [32, 16, 16]
-C.GRIPN.GD_LAYERS = [16, 32]
-C.GRIPN.DD_LAYERS = [sum(C.GRIPN.GD_LAYERS), 16]
-
-# ---------------------------------------------------------------------------- #
+C.SOLVER.MAX_EPOCHS = 66
+C.SOLVER.LOG_EVERY_N_STEPS = 1
+
+# ---------------------------------------------------------
+# GripNet supervertex configs
+# ---------------------------------------------------------
+# create configuration for each supervertex
+# Elements in `EXTER_AGG_CHANNELS_LIST` should be a list of
+# 	combinations of supervertex name (str) and embedding
+# 	dimension (int)
+# `MODE` is either "cat" or "add"
+
+C.GRIPN_SV1 = CfgNode()
+C.GRIPN_SV1.NAME = "gene"
```

Comment:
In the ReadMe, you talk about protein rather than gene, consider to revise there to be consistent with here, with some explanations if possible for new comers into this field.